ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ParticipantPruningIT.scala

```diff
@@ -529,132 +526,6 @@ class ParticipantPruningIT extends LedgerTestSuite {
     }
   })
 
-  test(
```

Comment:
Extracted these tests in their separate `MultiParticipantDivulgencePruningIT` so it can be run separately on a multi-participant test bed.